Dockerfile

```diff
@@ -19,14 +19,21 @@ COPY main.go main.go
 COPY api/ api/
 COPY controllers/ controllers/
 
+# The container running the operator/controllers needs the libvirt libraries as
+# as the peerpod-ctrl needs to dynamically link with libvirt
+RUN apt-get update -y && apt-get install -y libvirt-dev
+
 # Build
-RUN CGO_ENABLED=0 GOOS=linux go build -a -o manager main.go
+RUN CGO_ENABLED=1 GOOS=linux go build -tags=libvirt -a -o manager main.go
```

Comment:
all of these changes should be related to peerpod-ctrl commit no?

Reply:
Yes they are. I will move them to the other peerpod-ctrl commit.